### PR TITLE
Avoid issues if no default list is found

### PR DESCRIPTION
**Pull request by berndsi**
plus some minor cleanup

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -68,7 +74,7 @@ function showLoginPage(url, title) {
 
 function showLoginPrompt(tab, url) {
     return getCanonicalPageTitle(tab).then(title => {
-        document.getElementById('loginButton').onclick = (element) => showLoginPage(url, title);
```
Comment (mdholloway) [inline popup.js:71]: Does the onclick listener still work (on both browsers) with this change?

Comment (mdholloway) [reply popup.js:71]: Okay, I checked and it does. There's another instance of the same that could come out on line 104 as well.

**Review by mdholloway — CHANGES_REQUESTED** [review]

File: popup.js (hunk 1 above)
Comment (berndsi) [reply popup.js:71]: Not sure what you mean at line 104.

Comment (mdholloway) [reply popup.js:71]: Oops, sorry, I'm looking at code added locally in a later commit.

**Review by mdholloway — APPROVED** [review]

**Pull request merged**